bin/flightTracker.js

```diff
@@ -46,10 +49,21 @@ var options = yargs(process.argv.slice(2))
         describe:
           "The airport name as you know. For example John F. Kennedy airport",
         type: "string",
-      });
+      })
+
+      // Will we be passing an access_key to the API?
+
+      // .option("access_key", {
+      //   alias: "k",
+      //   description: "You AviationStack API access key",
+      //   type: "string",
+      //   demandOption: true
+      // })
     },
     (argv) => {
-      if (argv["name"]) {
+      if (argv['name']) {
+        // REQUIRES API KEY
+        // flightTracker.findIATACode(argv['name'])
```

Comment:
This functiona will require the API key. If the user hasn't passed the key, or they haven't fetched it from BitWarden, then this function needs to throw an error and prompt the user to pick one of the two approaches to getting the key.

Reply:
Now the flightTracker's methods check for an apiKey.